nemo/collections/asr/parts/submodules/rnnt_loop_labels_computer.py

```diff
@@ -14,19 +14,179 @@
 
 from typing import Any, Optional, Tuple
 
+import numpy as np
 import torch
 import torch.nn.functional as F
 from omegaconf import DictConfig
 
 from nemo.collections.asr.parts.utils import rnnt_utils
 from nemo.collections.asr.parts.utils.asr_confidence_utils import ConfidenceMethodMixin
+from nemo.core.utils.cuda_python_utils import (
+    assert_drv,
+    check_cuda_python_cuda_graphs_conditional_nodes_supported,
+    cu_call,
+    with_conditional_node,
+)
+from nemo.utils import logging
+
+try:
+    from cuda import cuda, cudart, nvrtc
+
+    HAVE_CUDA_PYTHON = True
+except ImportError:
+    HAVE_CUDA_PYTHON = False
+
+
+def run_nvrtc(kernel_string, kernel_name):
```

Comment:
I have no problem with moving this copy-pasted function into `core/utils/cuda_python_utils.py` FWIW.